lib/commands/config/write.ts

```diff
@@ -66,9 +66,8 @@ export default class ConfigWriteCmd extends Command {
 		help: cf.help,
 	};
 
-	public static authenticated = true;
-
 	public static root = true;
+	public static offlineCompatible = true;
```

Comment:
Strictly speaking, `config read/write/inject` will only be `offlineCompatible` after #2369 is merged. Until then, some dependencies make calls to balenaCloud API endpoints to retrieve device type details that contain partition numbers. But I suppose there is no harm in setting `offlineCompatible` already now, so no need to change it.

Reply:
#2369 has been merged, so that's no longer a concern.